workspace/src/main/kotlin/com/cosmotech/workspace/azure/WorkspaceServiceImpl.kt

```diff
@@ -191,18 +196,6 @@ internal class WorkspaceServiceImpl(
       hasChanged = true
     }
 
```

Comment:
Why are `users` not handled in the `updateWorkspace` method ? I think authorized users should also be able to replace the users list as part of a standard `PATCH` request on the Workspace.

Reply:
Refacto needed: Ownership tested at the begining of code: ownerId test no more needed.
=> Remove 'ownerId' and 'users' from mutation excluded fields for direct mutation.
FindSolutionById will throw 404 if not fount which is fine.

Reply:
done